README.md

```diff
@@ -1,22 +1,13 @@
 # Climate Informatics 2024 Artefact Evaluation
- 
-## The Problem
-<!-- Briefly describe what issue you are trying to address with this material -->
 
-## The Solution
-<!-- Briefly describe how your material fixes the problem! -->
+This repository describes the process for artefact evaluation (AE) at the
+Climate Informatics 2024 conference.
 
-## What are we doing?
-<!-- Describe the activities of people involved in this material. What have you done so far and what are you intending to do next. This section could include Usage instructions, describing how users use the material in this repository. -->
+Climate Informatics, like many other communities and fields, has software at its heart. Underlying most publications is a novel piece of software playing some critical role, e.g., embodying a model, processing or analysing data, or producing a visualisation. In order for such software artefacts to have the most impact, they should be available, functional, and reusable, such that other researchers can benefit from the work, verify the claims of the paper, and then build upon the software to do more great work. These ideals are summarised by the FAIR principles of data, which can be applied to software: research software should be Findable, Accessible, Interoperable, and Reusable. In order to help promote [FAIR software](https://www.nature.com/articles/s41597-022-01710-x), Climate Informatics is embarking, for the first time, on an _Artefact Evaluation phase_ for full paper submissions, after those submissions have been accepted for publication as the conference proceedings in [Environmental Data Science](https://www.cambridge.org/core/journals/environmental-data-science) following the traditional peer-review process. Artefact Evaluation provides an opportunity to embed the values of reproducibility into the publication process in a lightweight opt-in fashion, thus encouraging authors to make software available and the results of the paper reproducible.
```

Comment:
I recommend we mention previous work e.g. the Reproducibility Challenge hosted in CI2023. I'll replace the DOI with the position paper (when published). The paper proposes artefact evaluation as part of the vision of trust and reward in future CI conferences.

```suggestion
Climate Informatics, like many other communities and fields, has software at its heart. Underlying most publications is a novel piece of software playing some critical role, e.g., embodying a model, processing or analysing data, or producing a visualisation. In order for such software artefacts to have the most impact, they should be available, functional, and reusable, such that other researchers can benefit from the work, verify the claims of the paper, and then build upon the software to do more great work. These ideals are summarised by the FAIR principles of data, which can be applied to software: research software should be Findable, Accessible, Interoperable, and Reusable. In order to help promote [FAIR software](https://www.nature.com/articles/s41597-022-01710-x) and following the positive outcomes of the [2023 Reproducibility Challenge](https://doi.org/10.5281/zenodo.1037613), Climate Informatics is embarking, for the first time, on an _Artefact Evaluation phase_ for full paper submissions, after those submissions have been accepted for publication as the conference proceedings in [Environmental Data Science](https://www.cambridge.org/core/journals/environmental-data-science) following the traditional peer-review process. Artefact Evaluation provides an opportunity to embed the values of reproducibility into the publication process in a lightweight opt-in fashion, thus encouraging authors to make software available and the results of the paper reproducible.
```

Reply:
I think I'd like to put in a bit more structure back into the README, but will do this after this PR is merged :)